Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -276,19 +276,31 @@ public class HTTPClient {
         public var timeout: Timeout
         /// Upstream proxy, defaults to no proxy.
         public var proxy: Proxy?
+        /// Ignore TLS unclean shutdown error, defaults to `false`.
+        public var ignoreNIOSSLUncleanShutdownError: Bool
```

Comment:
=> `ignoreNIOSSLUncleanShutdownError` is a mouthful imo, maybe `ignoreUncleanSSLShutdown`? or something even shorter if someone can come up with a suggestion?

=> what are the argument for the default to be false vs. true?

Reply:
I’m in favour of a shorter name.

False is a good default because we already ignore it in the cases where it’s 100% safe to do so: setting this switch to true would ignore it in the cases where it potentially opens your application up to some theoretical risks.

Reply:
maybe `ignoreSSLUncleanShutdown`?

Reply:
`ignoreUncleanSSLShutdown` seem more natural?

Reply:
+1 for `ignoreUncleanSSLShutdown`

Reply:
@tomerd It was renamed.